aries-site/src/examples/templates/global-header/components/AppIdentity.js

```diff
@@ -0,0 +1,69 @@
+import React, { forwardRef, useContext } from 'react';
+import PropTypes from 'prop-types';
+import { Button, Box, ResponsiveContext, Text } from 'grommet';
+import { Aruba, Hpe } from 'grommet-icons';
+import { UserContext, SidebarLayer } from '.';
+
+const brands = {
+  hpe: {
+    name: 'HPE',
+    logo: <Hpe color="brand" />,
+  },
+  aruba: {
+    name: 'Aruba',
+    logo: <Aruba color="plain" />,
+  },
+};
+
+export const AppIdentity = forwardRef(
+  ({ brand, href, logoOnly, title, ...rest }, ref) => {
+    const { user } = useContext(UserContext);
+    const size = useContext(ResponsiveContext);
+
+    return (
+      <Box
+        align="center"
+        direction="row"
+        gap={size !== 'small' ? 'medium' : 'small'}
+      >
+        {user && <SidebarLayer />}
+        <Button href={href} ref={ref} {...rest} plain>
+          <Box
+            direction="row"
+            align="center"
+            gap="medium"
+            // pad maintains accessible hit target
+            // non-responsive maintains same dimensions for mobile
+            pad={{ vertical: 'small' }}
+            responsive={false}
+          >
+            {brand && user ? brands[brand].logo : brands.hpe.logo}
+            {!logoOnly && (
+              <Box direction="row" gap="xsmall" wrap>
+                {user ? (
```

Comment:
A code comment here would be useful to explain intent (e.g. If we do not have a signed in user, then there will be no related services. Show HPE and sign in screen).

Something like that would be helpful.

Reply:
Agreed, I will add a comment in.

Reply:
Done.